pkg/restic/backupper.go

```diff
@@ -158,11 +158,12 @@ func (b *backupper) BackupPodVolumes(backup *velerov1api.Backup, pod *corev1api.
 		}
 
 		volumeBackup := newPodVolumeBackup(backup, pod, volume, repo.Spec.ResticIdentifier, pvc)
-		numVolumeSnapshots++
 		if volumeBackup, err = b.repoManager.veleroClient.VeleroV1().PodVolumeBackups(volumeBackup.Namespace).Create(volumeBackup); err != nil {
 			errs = append(errs, err)
+			log.Errorf("Failed to create volumeBackup %s for volume %s in pod %s/%s: %v", volumeBackup.Name, pod.Namespace, pod.Name, err)
```

Comment:
We usually try not to both log *and* return errors, since that usually leads to double-logging. In this case I would just return it.